style/sass/.scss-lint.yml

```diff
@@ -12,7 +12,6 @@ linters:
 
   BemDepth:
     enabled: false
-    max_elements: 1
```

Comment:
I 👍  this. When https://github.com/sasstools/sass-lint/issues/413  started showing up in Hound comments, I didn't not like it. I understand some developers prefer only one level of BEM depth in order to allow enforce shorter class names, but I believe that is a band-aid that belies a strategic problem in the pattern itself. 

I prefer ancestral (non-direct descendant, just ancestral) embedding into BEM naming. The reason: it creates a balancing feedback loop for the developer to signal that if a class is too long, maybe there's more than one pattern here.